client/analytics/report/revenue/chart.js

```diff
@@ -0,0 +1,92 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component, Fragment } from '@wordpress/element';
+import { find } from 'lodash';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import ReportGraph from 'analytics/components/graph';
+import ReportSummary from 'analytics/components/summary';
+
+class OrdersReportChart extends Component {
+	constructor( props ) {
```

Comment:
Is this needed here since we aren't tracking any state or binding methods doing anything outside of calling `super`?